bigbluebutton-html5/imports/api/2.0/voice-users/server/handlers/joinVoiceUser.js

```diff
@@ -4,6 +4,7 @@ import addVoiceUser from '../modifiers/addVoiceUser';
 
 export default function handleJoinVoiceUser({ body }, meetingId) {
   const voiceUser = body;
+  voiceUser.joined = true;
```

Comment:
I think it's better to add property `joined` in addVoiceUser.js in the modifier
Also you won't be checking it in ` joined: Boolean, // This is a HTML5 only param.`